src/crypto/browser.ts

```diff
@@ -0,0 +1,122 @@
+import * as ed25519 from 'noble-ed25519'
+import rsaOperations from 'keystore-idb/rsa'
+import utils from 'keystore-idb/utils'
+import aes from 'keystore-idb/aes'
+import { CharSize, SymmKeyLength } from 'keystore-idb/types'
+
+import { assertBrowser } from '../common/browser'
+import * as keystore from '../keystore'
+
+
+export const encrypt = async (data: Uint8Array, keyStr: string): Promise<Uint8Array> => {
+  assertBrowser('aes.encrypt')
```

Comment:
Give user's better error messages if they try to call a browser-specific method in node